cpp/src/arrow/compute/api_aggregate.h

```diff
@@ -76,6 +76,18 @@ struct ARROW_EXPORT MinMaxOptions : public FunctionOptions {
   enum Mode null_handling = SKIP;
 };
 
+/// \brief Control Delta Degrees of Freedom (ddof) of Variance and Stddev kernel
+///
+/// The divisor used in calculations is N - ddof, where N is the number of elements.
+/// By default, ddof is zero, and population variance or stddev is returned.
+struct ARROW_EXPORT VarStdOptions : public FunctionOptions {
+  explicit VarStdOptions(int ddof = 0) : ddof(ddof) {}
```

Comment:
Naming suggestion: how about `DdofOptions`?

Reply:
I'm not sure if we will add options later, e.g., to ignore NaN

Reply:
I agree there might be other options in the future. But with the renaming of the functions, maybe call it `VarianceOptions` instead?

Reply:
Kind of struggling as stddev kernel uses this same option.
Maybe export both `VarianceOptions` and `StddevOptions` and alias them as same type internally?

Reply:
Only one is useful. Just `VarianceOptions` IMHO.